### PR TITLE
Add modules workflow

### PR DESCRIPTION
**Pull request by kdaud**
Description -> Add modules workflow

Ticket -> [RATEST-334](https://issues.openmrs.org/browse/RATEST-334)

### CODE REVIEW

File: qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ModulesSteps.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    	assertTrue(textExists("Modules"));
+    }
+
+    @And ("the user checks the list of started modules")
+    public void checkListOfStartedModules() {
+    	WebElement moduleListing = modulesPage.findElementById("moduleListing");
+    	List<WebElement> firstColumn = moduleListing.findElements(By.cssSelector("#moduleListing table tbody td"));
+    	for (WebElement eachModule : firstColumn) {
+            Assert.assertFalse(eachModule.getText().contains("moduleNotStarted"));
+        }
```
Comment (sherrif10) [inline qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ModulesSteps.java:71]: This last scenario seems uncomplete.  What happens when the user checks the list of started modules ?

Comment (kdaud) [reply qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ModulesSteps.java:71]: The test is asserting that all the modules on the page are well started. In short the test will fail if there is a module which is not started.

Comment (sherrif10) [reply qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ModulesSteps.java:71]: I understand that, However on the perspective of nontechnical folk, that last scenario sounds as if there is another scenario so having a feature file for at least confirming

Comment (kdaud) [reply qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ModulesSteps.java:71]: Ooh I see. So have made an adjustment in the last step checkout the recent commit have made and see if all is well now.

Comment (sherrif10) [reply qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/ModulesSteps.java:71]: Looks good.

**Review by sherrif10 — APPROVED** [review]

**Pull request merged**